diwako_cbrn.vr/scripts/cbrn/functions/fn_handleDamage.sqf

```diff
@@ -56,6 +68,10 @@ if (_actualThreat < 1) exitWith {
     cbrn_mask_damage ppEffectCommit 5;
 };
 
+// Cause the unit to have intoxication in KAT
+if (cbrn_kat_enabled) then {
+    _unit setVariable ["kat_chemical_airPoisoning", true];
+};
```

Comment:
Can this be also included in the first check if `cbrn_kat_enabled` is `true`? Will this `kat_chemical_airPoisoning` value ever be `false`?

Reply:
This variable is used to give the "intoxicated" medical condition to the player. Therefore, it is placed alone and behind ln 66 to ln 69.

Reply:
Fine by me. I do not really know anything how KAT works.